### DISCUSSION TITLE
How to initialize the deepxde model with a trained neural network?

### DISCUSSION THREAD

**Discussion by mynanshan** [lululxvi/deepxde#1718] [Q&A] [answered]
Suppose that I am doing an inverse problem. I want to train the deepxde model with some rough data first, and then continue updating the trained neural network with a new dataset.

Then the question comes down to, how can I use the outcome as the initialization in the second training stage, or equivalently, change the dataset in the middle.

In the current deepxde framework with `model = dde.Model(data, net)`, it seems that we have to create a new model object whenever switching to a new dataset.

**Comment by vl-dud** [comment, marked as answer]
```
model = dde.Model(first_data, net)
model.compile("adam", lr=0.001)
model.train()
model.data = second_data
model.compile("adam", lr=0.001)
model.train()
```
